src/routes/safe/components/Apps/components/ManageApps.tsx

```diff
@@ -2,8 +2,8 @@ import { ButtonLink, ManageListModal } from '@gnosis.pm/safe-react-components'
 import React, { useState } from 'react'
 
 import appsIconSvg from 'src/routes/safe/components/Transactions/TxsTable/TxType/assets/appsIcon.svg'
```

Comment:
can we remove this image and use the one from SRC? https://components.gnosis-safe.io/?path=/docs/data-display-icon--icons

Reply:
```
import AppsIcon from '@gnosis.pm/safe-react-components/dist/dataDisplay/Icon/images/apps.svg'
``` 
you mean this? ManageListModal doesn't accept an icon

Reply:
It only accepts an url for the image